pkgs/by-name/te/television/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "television";
+  version = "0.5.0";
+  src = fetchFromGitHub {
+    owner = "alexpasmantier";
+    repo = pname;
```

`repo = pname;` is an anti-pattern iirc.

```suggestion
    repo = "television";
```